### ISSUE TITLE
Dead link in documentation

### ISSUE THREAD

**Issue by eavraamsdonk** [Kruptein/planarally-docs#99]
On this page in the documentation "https://www.planarally.io/docs/dm/light-shadows/" the link "first session tutorial" points here "https://www.planarally.io/tutorial/in-game/first-session/", which gives a "404 - not found" page.

**Comment by Kruptein** [comment]
Good catch, that should be https://www.planarally.io/tutorial/dm/first-session/

**Comment by 10leej** [comment]
"https://www.planarally.io/learn/dm/first-session/" now points to "https://www.planarally.io/tutorial/out-of-game/asset-manager/" however the page for "https://www.planarally.io/tutorial/dm/first-session/" like previously mentioned no longer exists.  
I'll work on a PR to update the first session page after my own session tonight.
